traffic_ops/app/lib/API/Configs/ApacheTrafficServer.pm

```diff
@@ -2671,6 +2671,27 @@ sub remap_dot_config {
 			if ( $ds->{range_request_handling} == RRH_CACHE_RANGE_REQUEST ) {
 				$mid_remap{ $ds->{org} } .= " \@plugin=cache_range_requests.so";
 			}
+
+			#Add DS profile records.config entries as overrided ATS parameters
+			if ( defined( $ds->{'param'}->{'records.config'} ) ) {
+				foreach my $rc_entry ( keys %{ $ds->{'param'}->{'records.config'} } ) {
+					my @rec_params = split(' ', $rc_entry, 3);
+
+					#Check that the parameter name matches the records.config syntax
+					if ( $rec_params[0] eq 'CONFIG') {
+						my @rec_values = split(' ', $ds->{'param'}->{'records.config'}->{$rc_entry}, 2);
+
+						#Check parameter value matches records.config syntax
+						if ( {map{$_=>1}('INT','FLOAT','STRING')}->{$rec_values[0]}) {
```

Comment:
this is pretty cluttered and would make more sense to create that hash prior to the loop:
```
my %goodtype = map { $=>1 } qw{INT FLOAT STRING};
...
if ( $goodtype{$rec_values[0]}) {
...
}
```